iambic/config/wizard.py

```diff
@@ -351,7 +351,7 @@ def __init__(self, repo_dir: str):
         self._default_region = None
 
         asyncio.run(self.set_config_details())
-
+        check_and_update_resource_limit(self.config)
```

Comment:
this is what actually makes the difference.

Reply:
Awesome! As an alternative - what do you think about using this function in main.py under the "setup" function - this is the current pattern (we use it in import and git plan currently).

Reply:
@mdaue2 There is a slight chicken-and-egg problem. The function takes a config but when `setup` is run, we may have a config to pass to the function. Wizard actually bootstrap config based on business logic in `async def set_config_details(self):`. That's the reasoning why I placed `check_and_update_resource_limit(self.config)` after `self_config_details` has run to ensure we can use `self.config`.